ape_starknet/accounts/_cli.py

```diff
@@ -151,7 +145,6 @@ def _list(cli_ctx):
     if value
     else None,
 )
-@click.option("--keyfile", help="Import an existing key-file", type=Path())
 @click.option("--class-hash", help="The class hash of the account contract.")
 @click.option("--salt", help="The contract address salt used when deploying the contract.")
 def _import(cli_ctx, alias, network, address, keyfile, class_hash, salt):
```

Comment:
@dtdang have to also delete from args!

```suggestion
def _import(cli_ctx, alias, network, address, class_hash, salt):
```